app/models/shortener/shortened_url.rb

```diff
@@ -125,6 +126,19 @@ def to_param
 
   private
 
+  def self.lookup_by_token(token, additional_params, track)
+    shortened_url = ::Shortener::ShortenedUrl.unexpired.where(unique_key: token).first
```

Comment:
`::Shortener::ShortenedUrl.unexpired.where(unique_key: token).first` should be `::Shortener::ShortenedUrl.unexpired.find_by(unique_key: token)`

Reply:
@lifeiscontent, pretty sure `find_by` is not supported by Rails 3.  I'm fine with that personally, but the README states that Rails 3 is a target for this project (see https://github.com/jpmcgrath/shortener#dependencies)

Reply:
ah, ok, makes sense 👍